src/mongoc/mongoc-uri.c

```diff
@@ -42,7 +42,8 @@ struct _mongoc_uri_t
    char                   *database;
    bson_t                  options;
    bson_t                  credentials;
-   bson_t                  read_prefs;
+   bson_t                  read_prefs_tags;
```

Comment:
Do we actually need that?  It seems like we can get the read_prefs_tags from the read_prefs via mongoc_read_prefs_get_tags()